file_optical.c

```diff
@@ -168,10 +168,6 @@ static void fbo_handler_init(struct fbo_handler *h, struct tcmu_device *dev,
 
 static void fbo_handler_destroy(struct fbo_handler *h)
 {
-	if (h->thr) {
-		pthread_kill(h->thr, SIGINT);
-		pthread_join(h->thr, NULL);
-	}
 	pthread_cond_destroy(&h->cond);
```

Comment:
I am not sure I understood this comment.

The cancel_thread call in libtcmu will only handle threads allocated by libtcmu right? It will not handle threads created by file_* will it?